sdk/identity/azure-identity/azure/identity/aio/_credentials/azure_powershell.py

```diff
@@ -112,13 +119,29 @@ async def get_token_info(self, *scopes: str, options: Optional[TokenRequestOptio
         :raises ~azure.core.exceptions.ClientAuthenticationError: the credential invoked Azure PowerShell but didn't
           receive an access token
         """
+
+        # Check if claims challenge is provided
+        if options and options.get("claims"):
+            claims_value = options.get("claims")
+            raise CredentialUnavailableError(
+                message=f"Failed to get token. Run Connect-AzAccount --ClaimsChallenge {claims_value}"
+            )
+
         if sys.platform.startswith("win") and not isinstance(asyncio.get_event_loop(), asyncio.ProactorEventLoop):
             return _SyncCredential().get_token_info(*scopes, options=options)
         return await self._get_token_base(*scopes, options=options)
 
     async def _get_token_base(
         self, *scopes: str, options: Optional[TokenRequestOptions] = None, **kwargs: Any
     ) -> AccessTokenInfo:
+
+        # Check if claims challenge is provided
+        if options and options.get("claims"):
+            claims_value = options.get("claims")
+            raise CredentialUnavailableError(
+                message=f"Failed to get token. Run Connect-AzAccount --ClaimsChallenge {claims_value}"
```

Comment:
It should be Failed to get token. Run Connect-AzAccount -ClaimsChallenge {claims_value}